lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +100,15 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if not tokenized_document or not isinstance(tokenized_document, list):
+            raise ValueError('inappropriate type of input argument or input argument is empty')
+        bm25_vector = [0.0] * len(self._vocabulary)
+        bm25 = calculate_bm25(self._vocabulary, tokenized_document, self._idf_values,
+                              k1=1.5, b=0.75, avg_doc_len=self._avg_doc_len, doc_len=len(tokenized_document))
```

Comment:
Do not set arguments that are defaults in this function (`k1`, `b`).

Reply:
done